core/cap-0046-02.md

```diff
@@ -285,25 +248,6 @@ The security concerns from CAP-0046
 apply here as well.
 
 In addition to those concerns, this CAP does not provide validators with much
-control over contracts on the network. The only mechanism they have is blocking
-all contract creations and invocations, which should only be used in drastic
-situations. This CAP does not define a way for validators to block specific
-contracts.
-
-## Potential Future Work
-
-### Mutable contracts support
-
-While the contracts are immutable in this CAP, it's already possible to make
-them 'mutable' via proxy contracts (for example, contract A forwards its
-method calls to contract B and ID of contract B is stored in the data of
-the contract A).
-
-To further support mutation via the proxy pattern we could do the following:
-
-- Allow contract to modify its `SCS_LEDGER_KEY_CONTRACT_CODE` entry (as the
-  modification can only happen from within the contract, this would need to be
-  implemented in the first installed version of the contract)
-- Allow specifying a contract ID as the contract's source reference, so that the
-  contract would be guaranteed to have exactly the same implementation as the
-  referred contract without any additional code.
+control over contracts on the network. The only mechanism they have is disabling
```

Comment:
I think there are more mechanisms now, as it's possible to limit resources on multiple dimensions (e.g. disable large writes or constrain CPU etc.)

Reply:
I removed this section since it no longer applies and probably belongs in a different CAP.